### PR TITLE
Remove duplicate ToolsUI WebStart file

### PR DESCRIPTION
**Pull request by cwardgar**
Fixes [MPS-625937](https://andy.unidata.ucar.edu/esupport/staff/index.php?_m=tickets&_a=viewticket&ticketid=28029).

Previously, we had both `netCDFtools.jnlp` and `ToolsUI.jnlp`, and they only differed in the `application-desc` element. Rather than maintain both and try to keep them in sync, we're nuking `ToolsUI.jnlp` and programmatically modifying `netCDFtools.jnlp` as necessary.

I also simplified the other filtered resources that are included in thredds.war.

### CODE REVIEW

File: tds/build.gradle [hunk 1]
```diff
@@ -65,31 +68,75 @@ test {
...
+    File inputFile = rootProject.findProject(':ui').file("src/main/resources/netCDFtools.jnlp")
+    inputs.file inputFile
+    
+    File outputFile = file("$buildDir/tmp/ToolsUI.jnlp")
+    outputs.file outputFile
+    
+    doLast {
+        GPathResult jnlp = new XmlSlurper().parse(inputFile)
+        
+        // Add "<argument>{catalog}#{dataset}</argument>" to the "application-desc" node
```
Comment (cwardgar) [inline tds/build.gradle]: This tiny bit of Groovy code changes
```xml
<application-desc main-class="ucar.nc2.ui.ToolsUI"/>
```
to
```xml
<application-desc main-class="ucar.nc2.ui.ToolsUI">
  <argument>{catalog}#{dataset}</argument>
</application-desc>
```
Cool, right?

Comment (lesserwhirls) [inline tds/build.gradle]: 👍 - Groovy is awesome!

**Comment by lesserwhirls** [comment]
Looks nice to me. Would you mind updating the copyright blocks on the java files with the following?

~~~java
/*
 * Copyright (c) 1998-2017 University Corporation for Atmospheric Research/Unidata
 */
~~~

**Comment by lesserwhirls** [comment]
Oops...meant to comment, not close.

**Comment by cwardgar** [comment]
Done.

**Comment by lesserwhirls** [comment]
Thanks @cwardgar!

**Pull request merged**